src/Stratis.SmartContracts.CLR/AddressExtensions.cs

```diff
@@ -18,32 +18,43 @@ public static uint160 ToUint160(this string base58Address, Network network)
 
         public static Address ToAddress(this uint160 address)
         {
-            return Address.Create(address.ToBytes());
+            return CreateAddress(address.ToBytes());
         }
 
         public static Address ToAddress(this string address, Network network)
         {
-            return Address.Create(address.ToUint160(network).ToBytes());
+            return CreateAddress(address.ToUint160(network).ToBytes());
         }
 
         public static Address ToAddress(this byte[] bytes)
         {
             if (bytes.Length != Address.Width)
                 throw new ArgumentOutOfRangeException(nameof(bytes), "Address must be 20 bytes wide");
 
-            return Address.Create(bytes);
+            return CreateAddress(bytes);
         }
 
         public static Address HexToAddress(this string hexString)
         {
             // uint160 only parses a big-endian hex string
             var result = Extensions.HexStringToBytes(hexString);
-            return Address.Create(result);
+            return CreateAddress(result);
         }
 
         public static string ToBase58Address(this uint160 address, Network network)
         {
             return new BitcoinPubKeyAddress(new KeyId(address), network).ToString();
         }
+
+        private static Address CreateAddress(byte[] bytes)
```

Comment:
It looks like the Address.Create is not used anywhere after this change (apart from tests) Can we remove it?

Reply:
We can, but I didn't want to make any changes to the Stratis.SmartContracts NuGet package.

Reply:
ok